paths/git-trouble/fc-commitsucks.md

```diff
@@ -0,0 +1,93 @@
+   1. To begin editing the commit message(s), enter `git commit --amend`, and edit the commit message. After editing the commit message, close your editor.
+   1. If you are happy with the message you typed, enter: `git rebase --continue`. If you need to change your commit message again, enter: `git commit --amend` and repeat these steps. You will need to perform this process of `git commit --amend`, edit the commit message, `git rebase --continue`, until you have edited each commit you marked with an `e` during the initial stage of the `rebase`.
+   1. Once you have finished editing commit messages, enter: `git rebase --continue`, and the `rebase` will finish.
+   1. After finishing the `rebase`, enter `git push --force`.
+
+didnt-push: |
+  So you have a couple of options when it comes to fixing a bad commit message that, so first, we need to identify if:
+
+   > Am I trying to fix the **last** commit I made or commit a few commits back?
+
+  If you aren't fixing your last commit, skip down some. If you are fixing the **last** commit you made you can do the following:
+
+   1. Ensure you are on the correct `branch` and enter: `git commit --amend`.
+   1. Enter the desired commit message and close the text editor.
+
+  **BOOM** you just fixed your terrible commit message and now no one is the wiser. Congratulations!!!
+
+  If you aren't fixing your **last** commit, you can perform the following:
+
+   1. Ensure you are on the correct `branch` and enter: `git log --oneline`
+   1. Identify the SHA-1 hash code associated with the commit when **file 4** was added.
+   1. Enter the SHA-1 hash of the identified commit using the following: `git reset --soft SHA-1`, where SHA-1 is the SHA-1 of the commit for when **file 4** was added.
+   1. Remove **file6.md** from the Staging Area by entering: `git reset HEAD file6.md`.
+   1. Create a commit with **file5.md** by entering: `git commit -m "YOUR MESSAGE"`.
+   1. Add **file6.md** to the Staging Area by entering: `git add file6.md`.
+   1. Create a new commit for **file 6** by entering: `git commit -m "YOUR COMMIT MESSAGE"`.
+
+show-me-how:
+tell-me-why: |
+  ## Commit --amend
+  The `commit --amend` command enables you to modify the message of the _last_ commit you made. This can be helpful if you identify a spelling error or grammatical issue with the commit message you created. This command **only** works with the _last_ commit you made and you need to use other commands to modify commits made prior to your last commit.
+
+  ## Reset
+  Where `revert`, a command we use in other exercises was a command that provided a non-destructive method to clean up commit history and modify changes made to your repository, reset is a potentially destructive command. When using reset you are removing any reference to the commit in the repository history and the original copies of the files will no longer be available.
```

Comment:
I think this should say "whereas `revert`". We should also surround `reset` in backticks.